src/config.rs

```diff
@@ -41,6 +43,7 @@ fn parse_cli_args() -> Result<CliArgs, getopts::Fail> {
         "IPC channel name to communicate and control verso",
         "",
     );
+    opts.optflag("", "no-panel", "Launch Verso without control panel");
```

Comment:
argument and struct field name should be the same. If field is called `with_panel`, this argument should be too.

Reply:
Makes sense, I was trying to make the arg to not being a negative condition, I'll change the cli arg to `no_panel` and name the local variable in verso `with_panel`